packages/schematics/angular/e2e/index.ts

```diff
@@ -125,7 +125,7 @@ export default function (options: E2eOptions): Rule {
 
     if (options.projectRoot !== undefined) {
       newProjectRoot = options.projectRoot;
-      appDir = newProjectRoot;
+      appDir = options.projectRoot;
```

Comment:
Undo this change?

Reply:
TypeScript is complaining maybe I do `!` and disable it with tslint